### PR TITLE
Change Rating fields

### PR DESCRIPTION
**Pull request by elenirotsides**

### CODE REVIEW

File: src/rest/util.ts [hunk 1]
```diff
@@ -82,13 +82,30 @@ export const isAuthenticated = async (
...
+export const convertToNum = (val: string): 0 | 1 | null => {
+	if (val === "0") {
+		return 0;
+	} else if (val === "1") {
+		return 1;
+	} else {
+		return null;
+	}
+};
+
```
Comment (elenirotsides) [inline src/rest/util.ts:101]: At the time of this function's creation, I could not think of a better solution for converting the string 'bool' values to actual nums, so please please lmk if you have any suggestions to make this better!!

Comment (carpenterd777) [reply src/rest/util.ts:101]: how about:
``ts
const num = Number(val);
return num === NaN ? null : num;
``

File: src/rest/util.ts [hunk 2]
```diff
@@ -82,13 +82,30 @@ export const isAuthenticated = async (
...
 };
 
+/**
+ * Used to convert what is supposed to be a numerical representation of a boolean value
+ * since the request body sends everything as a string
+ * Returns 0, 1, or null
+ * @param val
+ * @returns numerical representation of a boolean value or null
+ */
+export const convertToNum = (val: string): 0 | 1 | null => {
```
Comment (carpenterd777) [inline src/rest/util.ts:92]:
```suggestion
export const convertToBinNum = (val: string): 0 | 1 | null => {
```

**Review by carpenterd777 — CHANGES_REQUESTED** [review]

**Comment by carpenterd777** [comment]
could you also make changes to the seed file?

**Comment by elenirotsides** [comment]
@carpenterd777 I wasn't finished and planned on updating all the tests and any other reference to the Rating object including the seed. I opened this as a draft to get an opinion on my approach towards refactoring the endpoints.

**Comment by carpenterd777** [comment]
ah I see, I took care of most of that in my PR if you'd like to check it out

File: __tests__/getNearbyPlaces.test.ts [hunk 3]
```diff
@@ -30,11 +30,15 @@ const mockPlace: GooglePlace = {
 
 const mockPlaceA11yData: Place = {
 	_id: mockPlace.place_id,
-	avgBraille: 5,
-	avgFontReadability: 5,
-	avgGuideDogFriendly: 5,
-	avgNavigability: 5,
-	avgStaffHelpfulness: 5,
+	guideDogAvg: 5,
+	isMenuAccessibleAvg: 5,
```
Comment (carpenterd777) [inline __tests__/getNearbyPlaces.test.ts:34]:
```suggestion
	isMenuAccessibleAvg: 1,
```
all of the `is` fields only should accept 0 or 1.

**Review by carpenterd777 — CHANGES_REQUESTED** [review]

File: __tests__/getNearbyPlaces.test.ts (hunk 3 above)
Comment (elenirotsides) [reply __tests__/getNearbyPlaces.test.ts:34]: whoops!! Thanks for catching that!

**Review by carpenterd777 — APPROVED** [review]

**Pull request merged**